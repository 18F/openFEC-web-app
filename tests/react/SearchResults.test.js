const rendered = require('react-test-renderer');
const React = require('react');
const ReactDOM = require('react-dom');
const SearchResults = require('../../static/js/legal/SearchResults');

it('renders search results', () => {
  const component = rendered.create(
    <SearchResults advisory_opinions={[]} q="" />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
