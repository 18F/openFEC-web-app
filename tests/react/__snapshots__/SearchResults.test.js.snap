// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders search results 1`] = `
<div
  className="message message--no-icon"
>
  <h2
    className="message__title"
  >
    No results
  </h2>
  <p>
    Sorry, we didnâ€™t find any documents matching 
    .
  </p>
  <div
    className="message--alert__bottom"
  >
    <p>
      Think this was a mistake?
    </p>
    <ul
      className="list--buttons"
    >
      <li>
        <a
          className="button button--standard"
          href="mailto:undefined"
        >
          Email our team
        </a>
      </li>
      <li>
        <a
          className="button button--standard"
          href="https://github.com/18f/fec/issues"
        >
          File an issue
        </a>
      </li>
    </ul>
  </div>
</div>
`;
